pkg/localstorev2/storer.go

```diff
@@ -370,6 +374,138 @@ func (p *putterSession) Done(addr swarm.Address) error { return p.done(addr) }
 
 func (p *putterSession) Cleanup() error { return p.cleanup() }
 
+func (db *DB) SubscribePush(ctx context.Context) (chunks chan swarm.Chunk, reset, stop func()) {
+	chunks = make(chan swarm.Chunk)
+	trigger := make(chan struct{}, 1)
+	resetC := make(chan struct{}, 1)
+
+	// send signal for the initial iteration
+	trigger <- struct{}{}
+
+	db.pushTriggersMu.Lock()
```

Comment:
let's make use of the ./pkg/localstore/internal/events pkg (it's in localstorev2-reserve branch atm)

Reply:
let's pair on this one, couldn't get far with it